docs/src/app/(public)/(content)/react/handbook/animation/page.mdx

```diff
@@ -73,4 +74,5 @@ Use the following Base UI attributes for creating CSS animations when a componen
 
 ## JavaScript animations
 
-JavaScript animation libraries such as [Motion](https://motion.dev) require control of the mounting and unmounting lifecycle of components. Most Base UI components are unmounted when hidden. These components usually provide the `keepMounted` prop to allow JavaScript animation libraries to take control.
+JavaScript animation libraries such as [Motion](https://motion.dev) require control of the mounting and unmounting lifecycle of components.
+Most Base UI components are unmounted when hidden. These components usually provide the `keepMounted` prop to allow JavaScript animation libraries to take control.
```

Comment:
I'm breaking the sentences into multiple lines so future PR diffs will be easier to review. It should have no impact on the rendered page.